blog/_src/posts/2016-10-31-meaningful-distinctions.md

```diff
@@ -0,0 +1,85 @@
+[other](https://www.jstor.org/stable/2589553)
+[uses](https://books.google.com/books?id=J4DkBwAAQBAJ&pg=PA6&lpg=PA6&dq=meaningful+distinctions+deserve+to+be+maintained&source=bl&ots=KYkrkBrJd_&sig=AAK1A_uIkQlVcYCY1TFljfA3CqA&hl=en&sa=X&ved=0ahUKEwjPwt2HmYbQAhWE6IMKHU5rB8YQ6AEIJTAC#v=onepage&q=meaningful%20distinctions%20deserve%20to%20be%20maintained&f=false)
+[of](https://books.google.com/books?id=oN5nsPkXhhsC&pg=PR6&lpg=PR6&dq=meaningful+distinctions+deserve+to+be+maintained&source=bl&ots=4doTufVdsy&sig=u3e_Z_xdN-tjt9p1eqQ88juA5Ns&hl=en&sa=X&ved=0ahUKEwjPwt2HmYbQAhWE6IMKHU5rB8YQ6AEIKDAD#v=onepage&q=meaningful%20distinctions%20deserve%20to%20be%20maintained&f=false)
+[the](https://books.google.com/books?id=GR44SKXCZJsC&pg=RA1-PA199&lpg=RA1-PA199&dq=meaningful+distinctions+deserve+to+be+maintained&source=bl&ots=lNpzR5QV7h&sig=IGg2Q_KtreSAhrbSJxsV7mQ8xok&hl=en&sa=X&ved=0ahUKEwjPwt2HmYbQAhWE6IMKHU5rB8YQ6AEIMDAF#v=onepage&q=meaningful%20distinctions%20deserve%20to%20be%20maintained&f=false)
+[quote](http://www.math.canterbury.ac.nz/php/groups/cm/faq/)
+[cited](http://www.ben-sherman.net/aux/evident-logic.pdf)
+a
+_Schizophrenia in comtemporary mathematics_, but I could not find an electronic copy.
+(It turns out, the AMS Bookstore page for [_Erret Bishop: Reflections on Him and His Research_](http://bookstore.ams.org/conm-39) includes a facsimile.)
+
+Lest anyone else is tempted to conjure the ancient magic of inter-library loan,
+here is a scan of the pages I borrowed. Thanks to the University of Toledo for
+supplying the hard copy.
+
+>   [prl.ccs.neu.edu/img/sicm.pdf](/img/sicm.pdf)
+
+The document is Bishop's "feeling for the philosophical issues involved" in constructive mathematics.
+First, Bishop lists "schizophrenic attributes" (trouble spots) of contemporary mathematics.
+Next, he gives basic principles of constructivism and Brouwer's interpretation of the logical quantifiers.
+Along the way, and as a source of examples, Bishop describes
+integers, sets, and real numbers.
+The emphasis is always on common-sense meaning and finite constructions.
+
+After a brief summary and reflection, the last ten pages list
+recent advances in constructive mathematics and upcoming tasks.
+The open tasks are particularly interesting:
+
+- systematically develop (constructive) algebra
+- give a constructive foundation for general topology
+- engage with the deeper "meaning of mathematics"
+
+The popular quote on "Meaningful Distinctions" appears early in the paper,
+as one of Bishop's four principles that "stand out as basic" to the philosophy
+of constructivism:
+
+> A. Mathematics is common sense.
+>
+> B. Do not ask whether a statement is true until you know what it means.
+>
+> C. A proof is any completely convincing argument.
+>
+> D. Meaningful distinctions deserve to be maintained.
+
+I had no idea that D. was "a principle", or that it had three siblings.
+
+To further tempt you into reading "the whole truth", here are some of my favorite phrases:
```

Comment:
Some of these phrases have periods and others don't. If it's in the middle of a sentence it should have ... on both sides, otherwise it should have a period if it ends a sentence